templates/bamboo/0/docker-compose.yml

```diff
@@ -0,0 +1,61 @@
+bamboo-data:
+    image: busybox
+    net: none
+    labels:
+      io.rancher.container.start_once: true
+    volumes:
+        - /home/bamboo
+
+bamboo-server:
+    image: matisq/bamboo-server:latest
+    ports:
+        - ${http_port}:8085
+        - ${agent_port}:54663
```

Comment:
Could you please use rancher lb's or traefik instead mapping host ports directly??